src/config.c

```diff
@@ -105,6 +106,13 @@ struct Keyword listener_stanza_grammar[] = {
             (int(*)(void *, char *))accept_listener_protocol,
             NULL,
             NULL},
+#ifdef SO_REUSEPORT
```

Comment:
The problem is that the platform sniproxy was built on and the platform it's being run can be different. And SO_REUSEPORT is only checked by CPP at build time. I've reworked my commits a little to indicate that (if) sniproxy was build without SO_REUSEPORT support if the config file contains "reuseport" directive.